contrib/hashicorp/vault/option.go

```diff
@@ -0,0 +1,39 @@
+package vault
+
+import "math"
+
+type config struct {
+	withAnalytics bool
+	analyticsRate float64
+	serviceName   string
+}
+
+// Option can be passed to NewHTTPClient and WrapHTTPClient to configure the tracer.
+// They can be created with the functions WithAnalytics, WithAnalyticsRate, and WithServiceName
```

Comment:
This line isn't needed. If you look at `godoc`, you will see that it groups all functions nicely below the type definition in the Table of Contents.

Reply:
👍